api/src/main/java/gov/va/vro/api/resources/VroResource.java

```diff
@@ -50,7 +52,11 @@ ResponseEntity<HealthDataAssessmentResponse> postHealthAssessment(
       description =
           "Posts health data for a specific patient and diagnostic code to launch evidence pdf generation")
   @PostMapping("/evidence-pdf")
-  @ResponseStatus(HttpStatus.CREATED)
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "201", description = "Successful Request"),
+        @ApiResponse(responseCode = "500", description = "PDF Generator Server Error")
```

Comment:
Can we also add 401 here for unauthorized? That will happen when there is no api key in the header.